src/localization/locales/es-ES.json

```diff
@@ -199,29 +199,23 @@
         "list_document_title": "Paisajes",
         "view_document_title": "{{name}}",
         "members_document_title": "Miembros: {{name}}",
+        "boundaries_document_title": "[TODO]Edit boundary: {{name}}",
         "list_column_actions_description": "Acciones",
         "view_map_title": "Mapa del paisaje",
-        "boundaries_title": "Actualiza la información geográfica del paisaje",
-        "boundaries_subtitle": "Sube un nuevo archivo",
-        "boundaries_description": "Muestra el límite del paisaje en un mapa cargando un archivo GeoJSON. Puedes obtener una vista previa antes de reemplazar el mapa actual.",
-        "boundaries_help_geojson": "¿No tienes un archivo GeoJSON? Siga estas sencillas instrucciones para crear uno.",
-        "boundaries_help_geojson_url": "https://terraso.org/help/draw-landscape-boundary",
-        "boundaries_select_title": "Selecciona un archivo y obten una vista previa",
+        "boundaries_title": "Actualiza la información geográfica del {{name}}",
+        "boundaries_help_geojson": "¿No tienes un archivo GeoJSON?",
+        "boundaries_help_geojson_detail": "[TODO]Siga estas sencillas instrucciones para crear uno.",
+        "boundaries_help_geojson_url": "[TODO]https://terraso.org/help/draw-landscape-boundary",
```

Comment:
Question: What is the needed change here? Right now there is no spanish content for that help section